sc-memory/sc-memory/sc_scs_helper.cpp

```diff
@@ -169,7 +169,26 @@ class StructGenerator
         {
           SetSCsGlobalIdtf(el.GetIdtf(), result);
         }
-        
+      }
+      else
+      {
+        ScType const & newType = el.GetType();
+        ScType const & oldType = m_ctx.GetElementType(result);
+        if (newType != oldType)
+        {
```

Comment:
Remove this if.

Reply:
I'm not sure that this action speed up the work of the builder. Will check.

Reply:
done